src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { push } from 'connected-react-router';
+
+import RenderInputTextField from '../RenderInputTextField';
+import RenderSelectField from '../RenderSelectField';
+import store from '../../data/store';
+import { AUDIT_TRACK, VERIFIED_TRACK, PROFESSIONAL_TRACK } from '../../data/constants';
+
+class CreateCourseForm extends React.Component {
+  getEnrollmentTrackOptions() {
+    return [
+      { label: 'Select enrollment track', value: '' },
+      { label: VERIFIED_TRACK.name, value: VERIFIED_TRACK.key },
+      { label: AUDIT_TRACK.name, value: AUDIT_TRACK.key },
+      { label: PROFESSIONAL_TRACK.name, value: PROFESSIONAL_TRACK.key },
+    ];
+  }
+
+  processOrganizations(organizations) {
+    const orgSelectList = [{ label: 'Select user organization', value: '' }];
+
+    if (organizations) {
+      organizations.forEach((org) => {
+        orgSelectList.push({ label: org.name, value: org.key });
+      });
+    }
+    return orgSelectList;
+  }
+
+  render() {
+    const {
+      handleSubmit,
+      organizations,
+      pristine,
+      submitting,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <div className="create-course-form row">
+          <div className="col">
+            <form onSubmit={handleSubmit}>
+              <Field
+                name="courseOrg"
+                component={RenderSelectField}
+                options={this.processOrganizations(organizations)}
+                label={
+                  <React.Fragment>
+                    Organization
+                    <span className="required">*</span>
+                  </React.Fragment>
+                }
+                required
+              />
+              <Field
+                name="courseTitle"
+                component={RenderInputTextField}
+                type="text"
+                label={
+                  <React.Fragment>
+                    Course title
+                    <span className="required">*</span>
+                  </React.Fragment>
```

Comment:
This feels like maybe it could be pulled into a component like "FormLabel" that takes a required parameter and adds the necessary asterisk or styling?

Does the "required" styling come from bootstrap?

Reply:
No, this was a style that was removed when copying the code over.  I think we can remove it and add style later, when we decide on the over all style of the pages, do you agree?

Reply:
Yeah, I don't have a strong opinion on whether we style it now or later. If you wanted red now without custom sass, you could replace `required` with `text-danger` which comes from bootstrap and gives a red color.

I do still like the idea of componentizing this. The color, whether the title is bolded, whether we use an asterisk, etc. are going to be the same across the app. Easier to pull it into its own thing.